### PR TITLE
General section, random tidies. Notes to bold italics

### PR DESCRIPTION
**Pull request by olliw42**
title says it

went through the pages in General section and streamlined it to current notation, and made some mroe tidies. Two main changes:

- changed Notes to bold & italics (as it was on the CLI page). On some pages the Notes are pretty relevant but do no stick out well, and  I got a feeeling some users miss them. So I wonder if we don't want to go to a bold&italics style.
- added a note one what bind and connect means.

### CODE REVIEW

File: docs/TROUBLE.md [hunk 1]
```diff
@@ -32,27 +32,27 @@ Carefully install your Tx module into your OpenTX or EdgeTX radio.  Some Tx modu
...
+- Be sure you have flashed the correct [firmware file](https://github.com/olliw42/mLRS#firmware-flashing).  Files which start with tx are only for modules functioning in the Tx role and files which start with rx are only for modules functioning in the receiver role.  Also, pay attention to the variants for different use cases and hardware configurations.  For example, there are 3 separate files for different hardware and use case options for the MatekSys mR900-30 when used as a Tx module and a 4-th file for when the same hardware is used as a receiver.  A module with rx firmware flashed won't connect with any other module with rx firmware flashed and tx firmware won't connect with tx firmware.
```
Comment (jlpoltrack) [inline docs/TROUBLE.md:45]: '4th' looks correct to me here.  Haven't seen '4-th' before.

**Comment by jlpoltrack** [comment]
Looks good to me - just one change.

File: docs/TROUBLE.md (hunk 1 above)
Comment (olliw42) [reply docs/TROUBLE.md:45]: okay ... I just find it very hard to read ... but I will revert :)

Comment (olliw42) [reply docs/TROUBLE.md]: grabbed the opportunity to also make all parameters being links (like most were)

Comment (jlpoltrack) [reply docs/TROUBLE.md]: I would probably prefer 'fourth' and then anything > 10 I would switch to 11th, 12th, etc.

**Comment by jlpoltrack** [comment]
Will merge a little later after final review.

**Pull request merged**